ocs_ci/utility/gcp.py

```diff
@@ -26,17 +26,16 @@
 
 # default location of files with necessary GCP cluster details
 SERVICE_ACCOUNT_KEY_FILEPATH = os.path.expanduser("~/.gcp/osServiceAccount.json")
-"""str: absolute filepath of json file with service account key
-
-This is json key file of ``sg-serv-account`` service account, which has full
-admin rights in given GCP project. The same key file is used by openshift
-installer during OCP installation to create all cluster resources from virtual
-machines to hostnames. Modules from ocs-ci are using the same key to get full
-cluster access as well.
-
-For more details, see `GCP documentation on ServiceAccountKey resource
-<https://cloud.google.com/iam/docs/reference/rest/v1/projects.serviceAccounts.keys>`_
-"""
+# str: absolute filepath of json file with service account key
+#
+# This is json key file of ``sg-serv-account`` service account, which has full
+# admin rights in given GCP project. The same key file is used by openshift
+# installer during OCP installation to create all cluster resources from virtual
+# machines to hostnames. Modules from ocs-ci are using the same key to get full
+# cluster access as well.
+#
+# For more details, see `GCP documentation on ServiceAccountKey resource
+# <https://cloud.google.com/iam/docs/reference/rest/v1/projects.serviceAccounts.keys>`_
```

Comment:
Would this change preserve this docstring in our sphinx build?

Right now, it's part of our documentation as expected: https://ocs-ci.readthedocs.io/en/latest/apidoc/ocs_ci.utility.html#ocs_ci.utility.gcp.SERVICE_ACCOUNT_KEY_FILEPATH

Reply:
Hmm, this I'm not sure of. Right now it's causing the following error in our `check-docstring-first` hook:
```
Check docstring is first.................................................Failed
- hook id: check-docstring-first
- exit code: 1

ocs_ci/utility/gcp.py:29 Multiple module docstrings (first docstring on line 2).
```

Reply:
I think we can check the docs from this build and resolve it if it looks good

Reply:
There is actually an open issue for this https://github.com/pre-commit/pre-commit-hooks/issues/159 with a potential fix that has yet to be merged. It may be worth reconsidering the use of `check-docstring-first` until that is fixed if we want to use attribute docstrings.

Reply:
Actually, looks like there may be a workaround for this: https://github.com/pre-commit/pre-commit-hooks/issues/159#issuecomment-559886109

Reply:
This is a bug in `check-docstring-first`, likely https://github.com/pre-commit/pre-commit-hooks/issues/159 - as you can see in the html sphinx build, both module docstring and variable docstring are present as expected.

I suggest to disable this particular hook.

Reply:
Yeah, I have tried to use the `#:` syntax workaround but it doesn't seem to work for me. I'm not seeing any attribute doc unless I go with the triple quote docstring method.

> I suggest to disable this particular hook.
I agree, I am going to revert the change to this docstring and disable the check entirely.